test/hummingbot/client/ui/test_style.py

```diff
@@ -52,16 +55,19 @@ def test_load_style_windows(self, is_windows_mock):
         global_config_map["logs-pane"] = self.ConfigVar("#FAFAFA")
         global_config_map["terminal-primary"] = self.ConfigVar("#FCFCFC")
 
-        style = Style.from_dict({"output-field": "bg:#ansigray #ansigray",
-                                 "input-field": "bg:#ansigray #ansiwhite",
-                                 "log-field": "bg:#ansigray #ansiwhite",
-                                 "header": "bg:#ansigray #ansiwhite",
-                                 "footer": "bg:#ansigray #ansiwhite",
-                                 "search": "#ansigray",
-                                 "search.current": "#ansigray",
-                                 "primary": "#ansigray",
+        style = Style.from_dict({"output-field": "bg:#ansiwhite #ansiwhite",
```

Comment:
@mhrvth Since we added #ansiwhite to the hex convert function, I am not sure if this is an expected result from this test, can you please help check?

Reply:
Strange, before there was ansiwhite in the test, so it looks like someone removed it from the hex_to_ansi()

Reply:
after further examination I conclude that it was really missing in the function